services/api/tests/routes/test_croissant.py

```diff
@@ -65,12 +69,23 @@ def test_get_croissant_from_dataset_infos() -> None:
     assert isinstance(croissant["recordSet"][0]["field"], list)
     assert isinstance(squad_info["features"], dict)
     assert len(croissant["recordSet"][0]["field"]) == len(squad_info["features"]) - 1
+    assert "1 skipped column: answers" in croissant["recordSet"][0]["description"]
```

Comment:
I don't get what is this `: answers"`. The string should contain

```
\n- 1 skipped column (max number of columns reached)
```

no?

Reply:
The dataset contains 5 features. But I noticed there's a skipped column for this dataset (hence the number `4` a bit below in the file, rather than the expected `5`). So I added a test here.

The message is inserted here: https://github.com/huggingface/datasets-server/blob/4d4b2f1e1163257664f707564bcfef337b4f947e/services/api/src/api/routes/croissant.py#L147

Reply:
`answers` corresponds to this feature: https://github.com/huggingface/datasets-server/blob/4d4b2f1e1163257664f707564bcfef337b4f947e/services/api/tests/routes/test_croissant.py#L15

Reply:
Oh OK, thanks! Sorry about that